bots/discord/atlbot/egg-a-t-l-bot.json

```diff
@@ -1,13 +1,15 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v2"
     },
     "exported_at": "2019-06-23T10:00:12-04:00",
     "name": "ATLBot",
     "author": "jfeldt19@gmail.com",
     "description": "ATLbot in ptero\r\n\r\nhttps:\/\/github.com\/ATLauncher\/discord-bot\/",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:alpine_nodejs-10",
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:alpine": "ghcr.io\/parkervcp\/yolks:alpine"
```

Comment:
This was specifically nodejs

Reply:
Yep, updated it to NodeJS 16 as that's their minimum supported version now.